src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py

```diff
@@ -6744,7 +6744,7 @@ def test_aks_create_edge_zone(self, resource_group, resource_group_location):
         self.test_resources_count = 0
         # kwargs for string formatting
         aks_name = self.create_random_name('cliakstest', 16)
-        kubernetes_version = "1.20.7"
+        kubernetes_version = "1.23.5"
```

Comment:
Just of curiosity, may I ask why do we need to upgrade this `kubernetes_version` ?

Reply:
Because version 1.20.7 is not supported in this region.